src/chocolatey/infrastructure.app/commands/ChocolateyInstallCommand.cs

```diff
@@ -208,7 +208,7 @@ public virtual void handle_additional_argument_parsing(IList<string> unparsedArg
 
             if (configuration.RegularOutput)
             {
-                warn_for_removed_options(unparsedArguments.Where(arg => arg.StartsWith("-")), _removedOptions);
+                warn_for_removed_options(unparsedArguments.Where(arg => arg.StartsWith("-")), _removedOptions, configuration.RegularOutput);

```

Comment:
Why are we passing in this parameter to the `warn_for_removed_options`? Why not simply not call this method at all?